cmd/rekor-cli/app/root_test.go

```diff
@@ -57,5 +57,5 @@ func TestAPIKey(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	_, _ = client.Tlog.GetPublicKey(nil)
+	_, _ = client.Pubkey.GetPublicKey(nil)
```

Comment:
I'm not terribly opinionated about the location, but would it be easier for backwards compat to keep it under client.TLog?

Reply:
This only changes the client bindings, not the actual REST API in any way; so it would be easier to leave it in Tlog... but I don't feel strongly about this.

Reply:
I might leave it this way then, since the public key isn't really associated with the tlog anymore. Happy to switch back if anyone prefers that!